apps/server/src/database/db-seeder.ts

```diff
@@ -121,189 +118,213 @@ export const seedDatabase = async (app: INestApplication) => {
     repeatEvery: 1,
   }).save();
 
-  // Const tasks
+  // Repeated chunks added manually, because repeat doesn`t work now
```

Comment:
Why repeat is not working now?

Reply:
Not really sure :\
From the calendar view, we can see only my custom-added repeated chunks.

Reply:
I'm not following.

The reason calendar is not rendering those, is because of the front. It has nothing to do with backend, so please, let's add repeat through normal taskService.createConst() method.